### PR TITLE
Updated Footer

### PR DESCRIPTION
**Pull request by Hey-Zeus-byte**
Hello! @fvaldez421 

Please review my code haha.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -70,32 +70,27 @@ const Footer = () => {
...
             </EmailLink>
           </p>
-          <div
+          <WithinFooterFirst
             style={{
-              // border: "2px solid red",
-              display: "flex",
-              justifyContent: "center",
-              alignItems: "baseline",
+              fontSize: "12px",
```
Comment (fvaldez421) [inline src/components/Footer.js]: This should be in the styled component

File: src/components/Footer.js [hunk 2]
```diff
@@ -70,32 +70,27 @@ const Footer = () => {
...
-        </FooterFlexWrapperFirst>
-        <FooterFlexWrapperSecond>
-          <strong
+          </WithinFooterFirst>
+          <WithinFooterFirstSecond
             style={{
-              display: "flex",
-              justifyContent: "center",
-              alignItems: "flex-end",
+              fontSize: "12px",
```
Comment (fvaldez421) [inline src/components/Footer.js]: Same comment as above

File: src/components/Footer.js [hunk 3]
```diff
@@ -70,32 +70,27 @@ const Footer = () => {
               Johnuxui@gmail.com
             </EmailLink>
           </p>
-          <div
+          <WithinFooterFirst
```
Comment (fvaldez421) [inline src/components/Footer.js]: These names are confusing. Not sure what first or second is supposed to mean

File: src/components/Footer.js [hunk 4]
```diff
@@ -55,41 +62,23 @@ const Footer = () => {
...
             <EmailLink href="mailto:Johnuxui@gmail.com">
               Johnuxui@gmail.com
             </EmailLink>
           </p>
-          <WithinFooterFirst
-            style={{
-              fontSize: "12px",
-            }}
-          >
+          <TextFontTypeOne>
```
Comment (fvaldez421) [inline src/components/Footer.js]: This naming is a little confusing bc you have an image in it. Can we come up with a better name?

File: src/components/Footer.js [hunk 5]
```diff
@@ -5,11 +5,8 @@ import {ContentWrapper} from "./ContentWrapper";
 const FooterContainer = styled.div`
   background-color: rgba(247, 245, 255);
   border-color: rgba(199, 199, 199);
-  /* border: 5px solid black; */
   display: flex;
-  flex-direction: row;
   justify-content: space-evenly;
```
Comment (fvaldez421) [inline src/components/Footer.js]: lets use the `space-between` so that our gutters are always even

**Comment by Hey-Zeus-byte** [comment]
For "Fixed footer container" I fixed the width size of the footer and updated the justify content in the following push.

Screen Shot 2022-08-22 at 11 52 20 PM

**Review by fvaldez421 — APPROVED** [review]

**Pull request merged**